orchestrator-salt/src/main/resources/salt/pillar/postgresql/disaster_recovery.sls

```diff
@@ -1,4 +1,2 @@
 disaster_recovery:
   object_storage_url:
-  aws_region: '' # empty string to prevent salt from passing the literal 'None'
```

Comment:
we can not really remove pillar property because then the older clusters will not work .

Reply:
Ok, I'll readd it and make a comment that it is not used anywhere.

There's _no_ way to remove a pillar property once it's been added? Not without breaking older clusters?

Reply:
We can remove it now since you just introduced it on master and it is not deployed anywhere. It only gets problematic if we would have released it.

Reply:
as @keyki  mentioned, this functionality is not yet released to customers so it this be fine.